packages/server-functions-vite-plugin/src/ast.ts

```diff
@@ -0,0 +1,20 @@
+import { parse } from '@babel/parser'
+import type { ParseResult } from '@babel/parser'
+
+export type ParseAstOptions = {
```

Comment:
we should really move this thing into a common package. we have this everywhere...